commit 40e119bd5577fc383990ad02b175ec64292f764b
Author: Jekyll Wu <adaptee@gmail.com>
Date:   Tue Oct 16 11:24:06 2012 +0900

    Fix a non-const lvalue build error with clang-3.1

    BUG=http://code.google.com/p/ibus/issues/detail?id=1506

    Review URL: https://codereview.appspot.com/6697048
    Patch from Jekyll Wu <adaptee@gmail.com>.

diff --git a/src/qibusserializable.h b/src/qibusserializable.h
index 594d973..92a32ed 100644
--- a/src/qibusserializable.h
+++ b/src/qibusserializable.h
@@ -91,7 +91,7 @@ private:
 
 template<typename T>
 QVariant &
-qVariantFromSerializable (const Pointer<T> &p, QVariant & variant = QVariant ())
+qVariantFromSerializable (const Pointer<T> &p, QVariant & variant)
 {
     QDBusArgument argument;
 
